src/App.js

```diff
@@ -1,16 +1,36 @@
-import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import { useState } from 'react';
 
 const App = () => {
+  const [entries, setEntries] = useState(chatMessages);
+  const [likes, setLikes] = useState(0);
```

Comment:
So organized, setting all the state variables at the top of the component! Love to see it 💖